app/celery/celery.py

```diff
@@ -167,7 +167,9 @@ def add_id_to_logger(task_id: str, task: Task, *args, **kwargs) -> None:
 
     task_name = getattr(task, 'name', 'UNKNOWN')
     # logger formatter includes notification_id if it is available
-    current_app.logger.debug('celery task_prerun task_id: %s | task_name: %s', task_id, task_name)
+    current_app.logger.debug(
+        '%s celery task_prerun task_id: %s | task_name: %s', task.request.hostname, task_id, task_name
```

Comment:
Wanted to identify if this was `priority` or `all` in the lowers, so we can be sure this works and continues to work in the future.